src/librustc_mir/build/mod.rs

```diff
@@ -341,6 +341,11 @@ impl<'a,'tcx> Builder<'a,'tcx> {
             }
         }
     }
+
+    pub fn check_overflow(&self) -> bool {
```

Comment:
@alexcrichton says he will bring up for @rust-lang/libs team discussion. 

@rust-lang/libs members, see comments above :)